rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/release/ReleaseController.java

```diff
@@ -1767,4 +1767,71 @@ public ResponseEntity<BulkOperationNode> bulkDeleteReleases(
+                            responseCode = "200", description = "License Data for the attachment.",
+                            content = {
+                                    @Content(mediaType = "application/json",
+                                            schema = @Schema(
+                                                    example = """
+                                                            {[
+                                                              "name": "License1",
+                                                              "text": "License text 1"
+                                                            ]}
+                                                            """
+                                            ))
+                            }
+                    ),
+                    @ApiResponse(
+                            responseCode = "400", description = "attachContentId cannot be parsed.",
+                            content = {
+                                    @Content(mediaType = "application/json",
+                                            schema = @Schema(
+                                                    example = """
+                                                            {[
+                                                              "error": "No applicable parser has been found for the attachment",
+                                                              "file": "source.zip"
+                                                            ]}
+                                                           """
+                                            ))
+                            }
+                    ),
+                    @ApiResponse(
+                            responseCode = "404", description = "Provided attachContentId not found."
+                    )
+            }
+    )
+    @GetMapping(value = RELEASES_URL + "/{id}/licenseData/{attachContentId}" )
+    public ResponseEntity<List<Map<String,String>>> getReleaseLicenseInfo(
+            @Parameter(description = "The ID of the release.")
+            @PathVariable("id") String relId,
+            @Parameter(description = "The ID of the attachment.")
+            @PathVariable("attachContentId") String attachContentId
+    ) throws TException {
+        User user = restControllerHelper.getSw360UserFromAuthentication();
+        Release sw360Release = releaseService.getReleaseForUserById(relId, user);
+        List<Map<String,String>> results = new ArrayList<>();
+        boolean found = false;
+        boolean type = true;
+        for (Attachment attachment : sw360Release.getAttachments()) {
+            if (attachContentId.equals(attachment.getAttachmentContentId())) {
+                found =true;
+                type = attachment.getAttachmentType() == AttachmentType.COMPONENT_LICENSE_INFO_XML
+                        || attachment.getAttachmentType() == AttachmentType.COMPONENT_LICENSE_INFO_COMBINED;
```

Comment:
Interesting thing I found, [LicenseInfoHandler](https://github.com/eclipse-sw360/sw360/blob/ab5c62292ea6b4c14106cf667ba13ab193eb0e6e/backend/licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/LicenseInfoHandler.java#L109-L113) has 3 parsers, `SPDXParser`, `CLIParser` and `CombinedCLIParser`. However, [getLicenseInfoForAttachment()](https://github.com/eclipse-sw360/sw360/blob/ab5c62292ea6b4c14106cf667ba13ab193eb0e6e/backend/licenseinfo/src/main/java/org/eclipse/sw360/licenseinfo/LicenseInfoHandler.java#L1053-L1060) only looks for `CLIParser` explicitly. Sometime in future, this has to be changed/fixed.